src/SFA.DAS.AssessorService.Web/Controllers/Apply/StandardController.cs

```diff
@@ -46,42 +49,59 @@ public async Task<IActionResult> Search(StandardViewModel model)
                 return RedirectToAction(nameof(Index), new { model.Id });
             }
 
-            var standards = await _apiClient.GetStandards();
+            var standards = await _standardVersionApiClient.GetAllStandardVersions();
 
-            model.Results = standards.Where(s => s.Title.Contains(model.StandardToFind, StringComparison.InvariantCultureIgnoreCase)).ToList();
+            model.Results = standards.Where(s => s.Title.Contains(model.StandardToFind, StringComparison.InvariantCultureIgnoreCase))
+                .GroupBy(
+                    s => s.IFateReferenceNumber,
+                    (key, stds)  => new StandardVersion()
+                    {
+                        IFateReferenceNumber = key,
+                        Title = stds.FirstOrDefault()?.Title
+                    })
+                .ToList();
 
             return View("~/Views/Application/Standard/FindStandardResults.cshtml", model);
         }
 
         [HttpGet("standard/{id}/confirm-standard/{standardCode}")]
         [ApplicationAuthorize(routeId: "Id")]
-        public async Task<IActionResult> ConfirmStandard(Guid id, int standardCode)
+        public async Task<IActionResult> ConfirmStandard(Guid id, string standardCode)
```

Comment:
I'd tweak the standardCode to StandardReference/IfateReferenceNumber now it's using the STxxx rather than lars code